app/src/main/res/values-si/string.xml

```diff
@@ -0,0 +1,1027 @@
+    <string name="accelerometer_description_text">\n\nඉහත රූපයේ දැක්වෙන්නේ ජංගම දුරකථනය කෙළින්ම රඳවාගෙන සිටින විට අක්ෂ තුනේම දිශාවයි.\n\nPSLab යෙදුමේ ත්වරණය මැනීමට පියවර:\n\n\u2022 ඉහත රූපයේ දැක්වෙන පරිදි උපාංගය රඳවා ගන්න.\n\n\u2022 ඕනෑම එක් අක්ෂයක් ‌හෝ  අක්ෂ කීීීීීීපයක් ඔස්සේ උපාංගය වේගවත් කරන්න.\n\n\u2022 කාඩ්පත්වල ඇති අගයන් හෝ කිසියම් විශේෂිත අක්ෂයක ලකුණු කළ ප්‍රස්ථාරය නිරීක්ෂණය කරන්න.\n\nචලනය වන වස්තුවක ත්වරණය මැනීමට ත්වරණමාන උපකරණය භාවිතා කළ හැකි අතර එම උපාංගය වස්තුව මතුපිට / ඇතුළත සවිකර, වස්තුව ත්වරණය කිිිිිිිිරීමෙන් සිදුකල හැකිය.\n\nසටහන: උපාංගය නිසියාකාරව සවිකර නොමැතිනම් වස්තුව වේගවත් නොකරන්න, නැතිනම් උපාංගයට හානි සිදුවිය හැක. </string>
+    <string name="no_accelerometer_sensor">උපාංගයට කිසිම ත්වරණමාන සංවේදකයක් නොමැත</string>
+
+    <string name="not_connected">සම්බන්ධකර නොමැත</string>
+    <string name="initialising_wait">Initialising Wait…</string>
+    <string name="device_connected_successfully">උපාංගය සාර්ථකව සම්බන්ධකර ඇත</string>
+    <string name="device_not_found">USB උපාංගය හමු නොවේ</string>
+    <string name="sensor_not_connected_tls">සංවේදකය සම්බන්ධකර නොමැත</string>
+
+    <string name="time_unit_la">කාලය (ms)</string>
+    <string name="view">දර්ශණය කරන්න</string>
+
+    <!-- Experiment Headers -->
+    <!-- Experiments -->
+
+
+    <string name="hmc5883l">hmc5883l</string>
+    <string name="mpu6050">mpu6050</string>
+    <string name="sht21">sht21</string>
+    <string name="tsl2561">tsl2561</string>
+    <string name="ads1115">ads1115</string>
+    <string name="mlx90614">mlx90614</string>
+    <string name="bmp180">bmp180</string>
+    <string name="mpu925x">mpu925x</string>
+
+    <string name="raw_data">අමු දත්ත</string>
+    <string name="configure">Configure</string>
+    <string name="plot">Plot</string>
+
+    <string name="data">දත්ත</string>
+    <string name="bx">Bx</string>
+    <string name="by">By</string>
+    <string name="bz">Bz</string>
+    <string name="ax">Ax</string>
+    <string name="ay">Ay</string>
+    <string name="az">Az</string>
+    <string name="gx">Gx</string>
+    <string name="gy">Gy</string>
+    <string name="gz">Gz</string>
+    <string name="temp">උෂ්ණත්වය</string>
+    <string name="luminosity">ආලෝක තීවෘතාවය</string>
+    <string name="humidity">ආර්ද්‍රතාවය</string>
+    <string name="object_temp">වස්තුවේ උශ්ණත්වය</string>
+    <string name="ambient_temp">පරිසර උෂ්ණත්වය</string>
+    <string name="temperature">උෂ්ණත්වය</string>
+    <string name="altitude">අක්ෂාංශ</string>
+    <string name="pressure">පීඩනය</string>
+    <string name="gyro_range">Set Gyro Range</string>
+    <string name="accel_range">Set Accel Range</string>
+    <string name="kalman">කාල්මාන් පෙරණය</string>
+    <string name="power_up">Power Up</string>
+    <string name="full_spectrum">Full spectrum</string>
+    <string name="infrared">අධෝරක්ත</string>
+    <string name="visible">පෙන්වන්න</string>
```

Comment:
දෘශ්‍ය

Reply:
I guess 'පෙන්වන්න' is okay for that. It is on a button.